benchmarks/playfair_cipher/oop_c#/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Text;
+using System.Drawing;
+using System.Text.RegularExpressions;
+using System.IO;
+
+namespace oop_c_
+{
+    class Program
+    {
+        static string TEST_STRING = File.ReadAllText("benchmarks/playfair_cipher/lines.txt");
+        static void Main(string[] args)
+        {
+            PlayFairCipher p = new PlayFairCipher("this is a great keyword");
+            string encrypt = p.Encrypt(TEST_STRING);
+            System.Console.WriteLine(encrypt.Length);
+            string decrypt = p.Decrypt(encrypt);

```

Comment:
```suggestion
            string decrypt = p.Decrypt(encrypt);
            System.Console.WriteLine(encrypt.Length);
```

Synes al IO skal være aller først of aller sidst